package.json

```diff
@@ -14,10 +14,14 @@
   },
   "dependencies": {
     "b64": "3.x.x",
-    "boom": "3.x.x",
+    "boom": "^4.1.0",
+    "code": "^4.0.0",
     "content": "3.x.x",
+    "form-data": "^2.1.1",
     "hoek": "4.x.x",
-    "nigel": "2.x.x"
+    "lab": "^11.1.0",
```

Comment:
This dependency is only used in `test/index.js` shouldn't it be a `devDependency`?